### PR TITLE
AsyncLoad and the Editor

### PR DESCRIPTION
**Pull request by mtias**
Implements AsyncLoad in different places in the editor and adjusts the base display when no placeholder is passed.

screen shot 2016-11-03 at 11 56 45

The post-schedule is a nice reduction:

screen shot 2016-11-03 at 12 07 09

### CODE REVIEW

**Comment by youknowriad** [comment]
This is really awesome 👍 

Everything is working properly but I noticed a small glitch when we open the PostSchedule Component by click on the date

async-load

**Comment by mtias** [comment]
Thanks for looking. Perhaps we can move the popover to the bottom-left. Any thoughts @retrofox ?

**Comment by retrofox** [comment]
When the `<Popover />` is rendered it's almost empty. I guess that `<div className="async-load" />` is rendered. Once the post-schdule is loaded it's mounted into the `<Popover />` component.

The issue is that the Popover is not able to update its position since there aren't properties which change they values, the state doesn't change and no events to update its position either.changes.

async-loader-popover-03

We could try to adjust the position through of css properties, covering all position cases, or find a way to notify to the `<Popover />` component that it should update its position.

**Comment by mtias** [comment]
What about just rendering it at the `bottom` instead of `right`?

File: client/post-editor/editor-action-bar/index.jsx [hunk 1]
```diff
@@ -68,7 +68,13 @@ export default React.createClass( {
 		return (
 			<div className="editor-action-bar">
 				<div className="editor-action-bar__first-group">
-					{ multiUserSite && <EditorAuthor post={ this.props.post } isNew={ this.props.isNew } /> }
+					{ multiUserSite &&
+						<AsyncLoad
```
Comment (aduth) [inline client/post-editor/editor-action-bar/index.jsx]: Nice thing about this is that the code for `<EditorAuthor />` will now never be loaded for single user sites 😄

**Comment by retrofox** [comment]
> What about just rendering it at the bottom instead of right?

That should fix this problem at one fell swoop :-D

**Comment by aduth** [comment]
Sorry, didn't think to account for schedule popover in marking "Ready to Merge". Unsure if you'd want to make that change first.

**Pull request merged**